rust/nasl-interpreter/src/built_in_functions/misc.rs

```diff
@@ -0,0 +1,117 @@
+// Copyright (C) 2023 Greenbone Networks GmbH
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+//! Defines NASL miscellaneous functions
+
+use std::{
+    fs::File,
+    io::{self, Read},
+};
+
+use sink::Sink;
+
+use crate::{error::FunctionError, ContextType, NaslFunction, NaslValue, Register};
+
+impl From<io::Error> for FunctionError {
```

Comment:
That is moved to error.rs and be deleted.

Reply:
Done